packages/design-system/src/components/Promo/Promo.test.js

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+
+import CloseIcon from 'react-material-icon-svg/dist/CloseIcon';
+
+import {Button} from '../Button';
+import Promo from './Promo';
+
+describe('Promo', () => {
```

Comment:
I would also add a test for render without a footer.

Reply:
✅